### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by Stepd3D** [KaylinOwO/Project-Apparatus#72]
Would Anti Kick be something doable?

**Comment by vOuii** [comment]
Done, waiting for merge

**Comment by vOuii** [comment]
> Would Anti Kick be something doable?

Done, waiting for merge.

**Comment by Stepd3D** [comment]
AntiKick hide players names now and call everyone #player until someone new join also affect the Player ESP

**Comment by lylyisbae** [comment]
No, it just takes a little bit for it to update. Just wait like 20 seconds for it to update.

**Comment by lylyisbae** [comment]
Also the reason behind this is because it breaks your players entity/steam name then waits for a callback from the host too to update offsets basically

**Comment by lylyisbae** [comment]
Can try adding a constant update feature but it may start making esp flicker constantly
